### DISCUSSION TITLE
Multiple "VS Code", "R for Windows", "Console Window Host" in task manager running

### DISCUSSION THREAD

**Discussion by werkstattcodes** [REditorSupport/vscode-R#1422] [Q&A]
For some reason, my windows task manager shows that my machine runs a large number of "Visual Studio Code", "R for Windows terminal front-end" and "Console Windows Host Processes". When starting VS code, there are 9 VS code processes; when opening a folder I have previously worked on is see in total 28 processes under the heading VS Code which consume more than 700 MB RAM. At this point I haven't triggered any R script at all.

I saw that there were some related issues filed on the vscode-Rs repo (e.g [here](https://github.com/REditorSupport/vscode-R/issues/1107)). I must admit that I didn't understand everything, but here are the steps which I took to tackle the issue:

- ~/.Renviron: R_LANGSVR_POOL_SIZE=1
- terminal.integrated.persistentSessionReviveProcess: never
- terminal.integrated.enablePersistentSessions: false

Any idea what I am missing, which steps I can take next? 
Many thanks

**Comment by eitsupi** [comment]
Maybe duplicate of #1107?
